examples/plotting/file/sprint.py

```diff
@@ -1,4 +1,6 @@
-# Based on http://www.nytimes.com/interactive/2012/08/05/sports/olympics/the-100-meter-dash-one-race-every-medalist-ever.html
+"""Based on http://www.nytimes.com/interactive/2012/08/05/sports/olympics/the-100-meter-dash-one-race-every-medalist-ever.html
```

Comment:
```suggestion
""" Based on http://www.nytimes.com/interactive/2012/08/05/sports/olympics/the-100-meter-dash-one-race-every-medalist-ever.html
```
Until we decide to change throughout, let's keep the formatting consistent